readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://<Kateryna199420>.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://<Kateryna199420>.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Need to fix the links and update it in the PR description
```suggestion
- [DEMO LINK](https://Kateryna199420.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://Kateryna199420.github.io/layout_search-bar-airbnb/report/html_report/)
```

Reply:
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/55200530/2874209c-0706-42bf-8e65-cad4e4ce3493)
Still not fixed, also update you PR description.